lib/action_cable/connection/identification.rb

```diff
@@ -27,7 +27,7 @@ def connection_identifier
 
       private
         def connection_gid(ids)
-          ids.map { |o| o.to_global_id.to_s }.sort.join(":")
+          ids.map { |o| (o.try(:to_global_id) || o.try(:to_s)).to_s }.sort.join(":")
```

Comment:
Why do we need to wrap `to_s` in `try`? If it's to ferret out nil values that's undone by the second `to_s`.

This should probably just be:

```ruby
ids.map { |o| (o.try(:to_global_id) || o).to_s }.sort.join(":")
```